PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
This file should get moved into the .github directory but I am not against having it in this repo